templates/taxi/car_detail.html

```diff
@@ -10,6 +10,14 @@ <h1>
     <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
       Update
     </a>
+    <form action="" method="post">
+      {% csrf_token %}
+      {% if user in car.drivers.all %}
+        <input type="submit" class="btn btn-danger link-to-page" value="Delete me from this car">
+      {% else %}
+        <input type="submit" class="btn btn-success link-to-page" value="Assign me to this car">
+      {% endif %}
+    </form>
```

Comment:
Code Quality issue: It seems like the form is not connected to any action. Make sure to specify the `action` attribute in the form to handle the submission properly.